encrypt/ecies/ecies.go

```diff
@@ -0,0 +1,113 @@
+// Package ecies implements the Elliptic Curve Integrated Encryption Scheme (ECIES).
+package ecies
+
+import (
+	"crypto/aes"
+	"crypto/cipher"
+	"crypto/sha256"
+	"errors"
+	"hash"
+
+	"github.com/dedis/kyber"
+	"github.com/dedis/kyber/util/random"
+	"golang.org/x/crypto/hkdf"
+)
+
+// Encrypt first performs a DH key exchange using the given public key, then
+// HKDF-derives a shared secret key (and nonce) from that, and finally encrypts
+// the given message using AES-GCM. Encrypt returns the ephemeral elliptic
+// curve point of the DH key exchange and the ciphertext or an error.
```

Comment:
last nitpick: now we need to precise that the last argument can be nil and it defaults to sha256 ;)

Reply:
Done.